plugins/core/IO/qLASIO/include/LasDetails.h

```diff
@@ -0,0 +1,356 @@
+#pragma once
+
+//##########################################################################
+//#                                                                        #
+//#                CLOUDCOMPARE PLUGIN: LAS-IO Plugin                      #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                   COPYRIGHT: Thomas Montaigu                           #
+//#                                                                        #
+//##########################################################################
+
+#include <CCTypes.h>
+#include <QtGlobal>
+
+#include <cmath>
+#include <limits>
+#include <string>
+#include <vector>
+
+class ccPointCloud;
+class ccScalarField;
+
+class QDataStream;
+
+struct laszip_header;
+struct laszip_vlr;
+typedef laszip_vlr laszip_vlr_struct;
+
+#define LAS_VLR_HEADER_SIZE 54
```

Comment:
could we switch to constexpr int or doubles?

Reply:
sure